typescript/vrotsc/src/compiler/transformer/fileTransformers/workflow.ts

```diff
@@ -515,10 +522,39 @@ export function getWorkflowTransformer(file: FileDescriptor, context: FileTransf
 
 		function buildAttributes(attributes: WorkflowParameter[]): void {
 			attributes.forEach(att => {
-				stringBuilder.append(`<attrib name="${att.name}" type="${att.type}" read-only="false" />`).appendLine();
+				stringBuilder.append(`<attrib name="${att.name}" type="${att.type}" read-only="false"`);
+				buildBindAttribute(att)
+				stringBuilder.append(` />`).appendLine();
 			});
 		}
 
+		function buildBindAttribute(att: WorkflowParameter) {
+			let value = "" + att.defaultValue;
+			if (att.bind) {
+				const index = value.lastIndexOf("/");
+				if (index == -1) {
+					throw new Error(`Invalid syntax for attribute "${att.name}" in workflow "${workflow.name}". It is specified that this value is bound to a Configuration (Elemene) but its value "${value}" `
+						+	`cannot be mapped to a Configuraiton Element path. No / separator in value. Expected formaat is "/Path/To/Confif/variable".`);
+				}
+				if (index >= value.length-1) {
+					throw new Error(`Invalid syntax for attribute "${att.name}" in workflow "${workflow.name}". It is specified that this value is bound to a Configuration (Elemene) `
+						+	`but its value "${value}" ends in / character. Expected formaat is "/Path/To/Confif/variable".`);
+				}
+				const key = value.substring(index+1).trim();
+				const path = value.substring(0, index).trim();
+				const id = context.configIdsMap[path];
+				if (id == null) {
```

Comment:
I think the check should be `if (id === null)`

Reply:
Fixed.